src/main/java/com/loofah/graph/api/queries/SkillQuery.java

```diff
@@ -0,0 +1,26 @@
+package com.loofah.graph.api.queries;
+
+import com.loofah.graph.api.models.Skill;
+import com.loofah.graph.api.repositories.SkillsRepository;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SkillQuery implements DataFetcher<Skill> {
```

Comment:
It also seems conventional to call classes like this "SkillDataFetcher" but I prefer "SkillQuery" - then we can have separate folders to keep query and mutation data fetchers.